iree/compiler/Translation/SPIRV/Kernels/CMakeLists.txt

```diff
@@ -23,3 +23,30 @@
 #    "reduce_untiled.comp"
 #   PUBLIC
 # )
+
+add_custom_command(
+  OUTPUT matmul.spv
+  COMMAND glslangValidator -V "${CMAKE_CURRENT_SOURCE_DIR}/matmul.comp" -o matmul.spv
+  DEPENDS glslangValidator
+)
+
+add_custom_command(
+  OUTPUT reduce_untiled.spv
+  COMMAND glslangValidator -V "${CMAKE_CURRENT_SOURCE_DIR}/reduce_untiled.comp" -o reduce_untiled.spv
+  DEPENDS glslangValidator
+)
+
+add_custom_command(
+  OUTPUT Kernels.h Kernels.cc
+  COMMAND generate_cc_embed_data matmul.spv reduce_untiled.spv
+          --output_header=Kernels.h
+          --output_impl=Kernels.cc
+          --identifier=Kernels
+          --cpp_namespace=mlir::iree_compiler::spirv_kernels
+          --flatten
+  DEPENDS generate_cc_embed_data
+          matmul.spv
+          reduce_untiled.spv
+)
+
+add_library(Kernels STATIC Kernels.cc)
```

Comment:
Can you change this name to `iree::compiler::Translation::SPIRV::Kernels` (or something equivalently namespaced)? Even as a step towards `iree_glsl_cc_library`, we should still keep the top level namespace clean.

Reply:
Also, sorry for the delay reviewing this, and thank you for your patience with the CLA process!

Reply:
Renamed `Kernels` to`iree_compiler_Translation_SPIRV_Kernels`, because of
```
CMake Error at iree/compiler/Translation/SPIRV/Kernels/CMakeLists.txt:52 (add_library):
  The target name "iree::compiler::Translation::SPIRV::Kernels" is reserved
  or not valid for certain CMake features, such as generator expressions, and
  may result in undefined behavior.
```

Further, no need to excuse. I think the delay was mostly caused on our side. On the contrary, thank you for your patience awaiting the signed CLA.

Reply:
Ah, yeah, I figured that error would appear. `iree_glsl_cc_library` should be able to use the `::` syntax if modeled after the other CMake libraries.